tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = py27,py34,py35,py36,pypy
+
+[testenv]
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+deps =
+    --no-binary
+     zc.recipe.egg
```

Comment:
I think it would be clearer if you did it on one line, with `--no-binary=zc.recipe.egg`.

It would be even less hacky if you used `install_command = pip install --no-binary=zc.recipe.egg {opts} {packages}`